pyvistaqt/rwi.py

```diff
@@ -0,0 +1,774 @@
+# modified from: https://gitlab.kitware.com/vtk/vtk/-/blob/master/Wrapping/Python/vtkmodules/qt/QVTKRenderWindowInteractor.py
```

Comment:
A comment about the relationship between this and the Mayavi version would be good if possible. But it could be a follow-up PR if someone wants... in which case please just add a TODO comment like `# TODO: Mayavi has a potentially different version that might be better or worse` so it's clear that we don't know which is better, and chose the VTK one just because it's the most obvious/upstream choice.